app/uk/gov/hmrc/essttpstubs/controllers/EligibilityController.scala

```diff
@@ -47,23 +47,23 @@ class EligibilityController @Inject() (
 
   def retrieveEligibilityData: Action[EligibilityRequest] = Action.async(parse.json[EligibilityRequest]) { implicit request =>
     LoggingHelper.logRequestInfo(logger  = logger, request = request)
-    val maybeResponse: Option[Status] = request.body.idValue match {
-      case "NotFound" => Some(NotFound)
-      case "500Error" => Some(InternalServerError)
-      case "502Error" => Some(BadGateway)
-      case "503Error" => Some(ServiceUnavailable)
-      case "504Error" => Some(GatewayTimeout)
-      case "422Error" => Some(UnprocessableEntity) // de-registered user response from ttp
-      case "499Error" => Some(new Status(499)) // we see some 499's in prod, as far as we can tell they're similar to 502
-      case _          => None
+    val maybeResponse: Option[Status] = request.body.identification.headOption.map(_.idValue.value) match {
+      case Some("NotFound") => Some(NotFound)
+      case Some("500Error") => Some(InternalServerError)
+      case Some("502Error") => Some(BadGateway)
+      case Some("503Error") => Some(ServiceUnavailable)
+      case Some("504Error") => Some(GatewayTimeout)
+      case Some("422Error") => Some(UnprocessableEntity) // de-registered user response from ttp
+      case Some("499Error") => Some(new Status(499)) // we see some 499's in prod, as far as we can tell they're similar to 502
+      case _                => None
     }
     maybeResponse.fold {
       eligibilityService.eligibilityData(request.body).flatMap {
         case Some(value) =>
           LoggingHelper.logResponseInfo(uri          = request.uri, logger = logger, responseBody = Json.toJson(value))
           Future.successful(Ok(Json.toJson(value)))
         case None =>
-          Future.successful(Ok(Json.toJson(EligibilityService.defaultEligibleResponse(RegimeType(request.body.regimeType), request.body.idValue))))
+          Future.successful(Ok(Json.toJson(EligibilityService.defaultEligibleResponse(RegimeType(request.body.regimeType), request.body.identification.headOption.map(_.idValue.value).getOrElse("")))))
```

Comment:
under the hood `defaultEligibilityResponse` is doing this currently:
```scala
def defaultEligibleResponse(regimeType: RegimeType, taxId: String): EligibilityCheckResult = {
    val identification: List[Identification] = regimeType match {
      case RegimeType("PAYE") =>
        List(Identification(IdType("EMPREF"), IdValue(taxId)), Identification(IdType("BROCS"), IdValue(taxId)))
      case RegimeType("VATC") =>
        List(Identification(IdType("VRN"), IdValue(taxId)))
      case RegimeType("SA") =>
        List(Identification(IdType("UTR"), IdValue(taxId)))
      case _ => // just default to epaye, we don't really care...
        List(Identification(IdType("EMPREF"), IdValue(taxId)), Identification(IdType("BROCS"), IdValue(taxId)))
    }
...
```
What might be better is to pass in the full list of identifiers to this method and then map it directly to the response type except for the case of PAYE where we add the `BROCS` id type as well. In the case of PAYE and VAT we could make sure we get one ID only and throw an exception otherwise to make it easier. This will mean that we get back a list of identifiers in the eligibility response for SA which is more likely what they're actually gonna do.